salt/cloud/clouds/azurearm.py

```diff
@@ -852,7 +852,7 @@ def create_network_interface(call=None, kwargs=None):
                     ]
                     break
             except CloudError as exc:
-                log.error('There was a cloud error: %s', exc)
+                log.error('There was a cloud error: {0}'.format(exc))
```

Comment:
please change this back to the other format, the previous method is faster.

Reply:
as you wish